hs-bindgen/src/HsBindgen/Backend/PP/Render.hs

```diff
@@ -9,16 +9,20 @@ module HsBindgen.Backend.PP.Render (
   , renderIO
   ) where
 
+import Data.Maybe (maybeToList)
```

Comment:
FYI, `maybeToList` is the same as `Foldable.toList` we have in `HsBindgen.Imports`.